spec/indicators_spec.py

```diff
@@ -32,7 +32,7 @@
             assert isinstance(profile, ProfilePVPC20A)
             data = profile.get(self.start_date, self.end_date)
             expect(data['indicator']['short_name']).to(
-                equal(u'Tarifa 2.0A (peaje por defecto)')
+                equal(u'Tarifa 2.0.A (peaje por defecto)')
```

Comment:
2.0A is correct name

Reply:
Yes, but the label used here is "2.0.A". I do not know why, but... this is ***** :hankey: